restart_process/main.go

```diff
@@ -0,0 +1,44 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"log"
+	"os"
+	"os/exec"
+	"strings"
+	"syscall"
+)
+
+var defaultWatchFile = "/.restart-proc"
+var watchFile = flag.String("watchfile", "", "File that entr will watch for changes; changes to this file trigger entr to rerun the command(s) passed")
+
+func main() {
+	flag.Parse()
+
+	args := os.Args[1:]
```

Comment:
I think you can pass `"/.restart-proc"` directly, as second argument for the default value in `flag.String` above, use `flag.Args()` below, and delete these 7 lines of code.

Reply:
ha yeah i just found `flag.Args()`. good call.